backend/internal/server/stream.go

```diff
@@ -215,13 +215,10 @@ func (ws *RpcWebsocket) WebsocketHandler(server *Server) httprouter.Handle {
 					continue MessageLoop
 				}
 
-				if found {
-					req.SendRaw("error", "'id' field used previous ID value")
-					continue MessageLoop
-				}
+				req.Context, req.cancel = context.WithCancel(ctx)
```

Comment:
Since unused contexts don't really clean themselves up, this should be created as late as possible